lstchain/calib/camera/calibrator.py

```diff
@@ -191,12 +190,17 @@ def _calibrate_dl1(self, event, telid):
         #      - gain selection before charge integration
 
 
-        no_gain_selection= np.zeros((waveforms.shape[0],waveforms.shape[1]), dtype=np.int)
+        no_gain_selection = np.zeros((waveforms.shape[0], waveforms.shape[1]), dtype=np.int)
+
+        charge = np.zeros(waveforms.shape[0])
+        peak_time = np.zeros(waveforms.shape[0])
```

Comment:
I am getting an error and I suspect it is here... charge[i] and peak_time[i] will later contain each an array with one entry per pixel. Why this initialization??

Reply:
This seems to work:
        charge = np.zeros(waveforms.shape[0]*waveforms.shape[1], dtype='float32')                                                                              
        peak_time = np.zeros(waveforms.shape[0]*waveforms.shape[1], dtype='float32')                                                                           

Did it work for anyone with the other initialization?

Reply:
It shouldn't have. (n_gains, n_pixels) is the correct shape here, here it's just a two element array